File: Codewars/Growth of a Population.py
import math
def nb_year(p0, percent, aug, p):
    counter = 0
    nw_pop = p0

    while nw_pop < p:
        nw_pop = math.floor(nw_pop)
        nw_pop = nw_pop + (nw_pop * (percent/100)) + aug
        counter += 1
    return int(counter)

File: Day 93/Homework/Growth of a Population.js
function nbYear(p0, percent, aug, p) {
    let years = 0;
    for (years; p0 < p; years++) {
      p0 += p0 * (percent / 100) + aug;
    }
    return years;
  }
  
  
  
  function nbYear(p0, percent, aug, p, years = 0) {
    return p0 >= p
      ? years
      : nbYear(
          Math.floor(p0 + p0 * (percent / 100)) + aug,
          percent,
          aug,
          p,
          years + 1
        );
  }